website/content/docs/oss/configuration/worker/pki-worker.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+page_title: PKI Worker Configuration
+description: |-
+  PKI worker-specific parameters.
+---
+
+
+## PKI Worker Configuration
+PKI Workers authenticate to Boundary using a new certificate-based method, allowing for worker deployment
+without using a shared KMS. PKI Workers can also be configured using CLI commands.
+
+PKI Workers require an accessible directory, defined by `auth_storage_path`, for credential storage.
+Example (not safe for production!):
+
+```hcl
+worker {
+  auth_storage_path="/tmp/boundary/demo-worker-1"
```

Comment:
I wouldn't use a path in `/tmp` as an example. Outside of dev mode where it's done automatically you'd never really want to do this because the creds will get lost whenever tmp is wiped, e.g. on a reboot.

Reply:
should we just recommend `/boundary/` instead?

`auth_storage_path="/boundary/demo-worker-1"`